src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx

```diff
@@ -169,6 +169,53 @@ class ViewPaymentsLists extends Component {
               you'll get a double payment. If this happens, VA Debt Management
               Center will contact you about collection.
             </p>
+            <h3 className="vads-u-font-size--h2">
+              What if I need to change my direct deposit or contact information?
+            </h3>
+            <p className="vads-u-margin-bottom--3">
+              Any changes you make in your profile will update across your
+              disability compensation, pension, claims and appeal, VR&E, and VA
+              health care benefits.
+            </p>
+            <p className="vads-u-margin-bottom--3">
+              <a href="/profile">
+                Go to your profile to make updates to your contact and direct
+                deposit information.
+              </a>
+            </p>
+            <h3 className="vads-u-font-size--h2">
+              What if I’m missing a payment?
+            </h3>
+            <p className="vads-u-margin-bottom--3">
+              Please wait 3 business days (Monday through Friday) before
+              contacting us to report that you haven’t received a payment. We
+              can’t trace payments before then.
+            </p>
+            <p className="vads-u-margin-bottom--3">
+              {' '}
+              To report a missing payment, contact us at{' '}
+              <Telephone contact={CONTACTS.VA_BENEFITS} />. Please have the
+              following information ready for the call: your address, Social
+              Security number or VA claim number. If you receive payments
+              through direct deposit, you’ll need your bank account information
+              too.
+            </p>
+            <h3 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary">
```

Comment:
should this be an `h2` and have an `h3` font size?

Reply:
@Mottie Nope, that would give us two H2's but based on the heirarchy of the information there should only be one.